helm-charts/helm2/strimzi-kafka-operator/values.yaml

```diff
@@ -117,7 +116,7 @@ readinessProbe:
   periodSeconds: 30
 
 # Override the docker image repository used by all Strimzi images
-# imageRepositoryOverride: foobar
+imageRepositoryOverride: ""
```

Comment:
`imageRepositoryOverride` and `imageTagOverride` are used in the chart but not present in the `values.yaml`. This was added to both charts for reference.